core/src/main/java/org/mobilitydata/gtfsvalidator/notice/DuplicatedColumnNotice.java

```diff
@@ -18,7 +18,11 @@
 
 import com.google.common.collect.ImmutableMap;
 
-/** The input file CSV header has the same column name repeated. */
+/**
+ * The input file CSV header has the same column name repeated.
+ *
+ * <p>Severity: {@code SeverityLevel.INFO}
```

Comment:
Should be an WARNING? The spec doesn't explicitly require this, although it would be a big headache for consumers, as they wouldn't know which column to use.